HW9.md

```diff
@@ -0,0 +1,181 @@
+    inet 192.168.225.128/24 brd 192.168.225.255 scope global noprefixroute dynamic ens33
+       valid_lft 1799sec preferred_lft 1799sec
+    inet6 fe80::dfdb:481:367b:712/64 scope link noprefixroute
+       valid_lft forever preferred_lft forever
+```
+
+   #### 1.3 using nmcli utility (statefull)
+
+```bash
+[hero@centos ~]$ sudo nmcli con mod ens33 +ipv4.addresses "192.168.225.129/24"
+[hero@centos ~]$ sudo reboot
+[hero@centos ~]$ ip a
+1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
+    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
+    inet 127.0.0.1/8 scope host lo
+       valid_lft forever preferred_lft forever
+    inet6 ::1/128 scope host
+       valid_lft forever preferred_lft forever
+2: ens33: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
+    link/ether 00:0c:29:dd:4d:e3 brd ff:ff:ff:ff:ff:ff
+    inet 192.168.225.128/24 brd 192.168.225.255 scope global noprefixroute dynamic ens33
+       valid_lft 1795sec preferred_lft 1795sec
+    inet 192.168.225.129/24 brd 192.168.225.255 scope global secondary noprefixroute ens33
+       valid_lft forever preferred_lft forever
+    inet6 fe80::dfdb:481:367b:712/64 scope link noprefixroute
+       valid_lft forever preferred_lft f
+```
+
+### 2. You should have a possibility to use ssh client to connect to your node using new address from previous step. Run tcpdump in separate tmux session or separate connection before starting ssh client and capture packets that are related to this ssh connection. Find packets that are related to TCP session establish.
+### 3. Close session. Find in tcpdump output packets that are related to TCP session closure.
+
+```bash
+[hero@HeroWithin ~]$ sudo tcpdump -i eth0 -w /home/hero/ch9/dump
+tcpdump: listening on eth0, link-type EN10MB (Ethernet), capture size 262144 bytes
+^C62 packets captured
+64 packets received by filter
+0 packets dropped by kernel
+==================================================================================
+[hero@HeroWithin ~]$ ssh hero@192.168.225.128
+hero@192.168.225.128's password:
+Last login: Sat Jan  1 20:36:54 2022 from 192.168.225.1
+[hero@centos ~]$ exit
+logout
+Connection to 192.168.225.128 closed.
+[hero@HeroWithin ~]$
+```
+
+#### terminal (Windows)
+
+##### Start SSH session (establishing a connection) ![start SSH session](https://user-images.githubusercontent.com/33420376/147834440-f2545088-2d76-4fe5-b019-053049dd1df5.png)
+##### The end SSH session ![closing a connection](https://user-images.githubusercontent.com/33420376/147834474-f4227143-3ec2-406a-b3d4-c74bcfb16706.png)
+
+### 4. run tcpdump and request any http site in separate session. Find HTTP request and answer packets with ASCII data in it.  Tcpdump command must be as strict as possible to capture only needed packages for this http request.
+
+#### Dump session
+
+```bash
+[hero@HeroWithin ~]$ sudo tcpdump -A tcp port 80 -i eth0 -w /home/hero/ch9/dump2
```

Comment:
`Tcpdump command must be as strict as possible to capture only needed packages for this http request.`
In that case the command will catch all `port 80`, `tcp`, `eth0` requests, not just `curl http://manpages.ubuntu.com/manpages/bionic/ru/man1/vimtutor.1.html`